exporters/metrics/src/main/java/com.google.cloud.opentelemetry.metric/MetricTranslator.java

```diff
@@ -31,6 +31,8 @@
   static final Set<String> KNOWN_DOMAINS = ImmutableSet
       .of("googleapis.com", "kubernetes.io", "istio.io", "knative.dev");
   static final long NANO_PER_SECOND = (long) 1e9;
+  static final String METRIC_DESCRIPTOR_TIME_UNIT = "ns";
+  static final String METRIC_DESCRIPTOR_DESCRIPTION = "OpenTelemetry metric";
```

Comment:
This seems to say nothing, since we are using OpenTelemetry SDK 🙂 
Why do we need to set it?
If it's required by mock server, we may set it in tests and not in production?

Reply:
Yep it's required by the mock server. I've changed this to use `metric.getDescriptor().getDescription()`